CharlieBackend.Core/Entities/Schedule.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.ComponentModel.DataAnnotations;   
+
+namespace CharlieBackend.Core.Entities
+{
+    public partial class Schedule : BaseEntity
+    {
+        public TimeSpan LessonStart { get; set; }
+
+        public TimeSpan LessonEnd { get; set; }
+
+        public long StudentGroupId { get; set; }
+
+        public virtual StudentGroup StudentGroup { get; set; }
+
+        public virtual RepeatRate RepeatRate { get; set; }
+
+        [Range(1, 31)]   
```

Comment:
Not all of the month have 31 days per month. Will it be handled somehow if user puts 31th day in October month, for example ?

Reply:
Yes, probably in service